src/reference/job-script-vars/var-list.rst

```diff
@@ -0,0 +1,119 @@
+CYLC_VERSION
+    Version of cylc installation used
+
+CYLC_ENV_NAME
+    Environment from which cylc commands run
+
```

Comment:
Suggest leaving this one undocumented:

```suggestion
```

Reason being:

* This variable may, or may not be set depending on how Cylc is deployed because it is set by the wrapper script, not by Cylc itself.
* It's probably a bad idea to use this in your workflows (i.e. don't write your tasks into the Cylc environment).